appcli/variables_manager.py

```diff
@@ -95,34 +98,119 @@ def set_all_variables(self, variables: Dict):
         self.__save(variables)
 
     def __get_configuration(self) -> Dict:
-        """Get the current configuration from file
+        """Get the current configuration from file(s)
 
         Returns:
             Dict: the current configuration
+
         """
-        try:
-            raw_data = self.configuration_file.read_text(encoding="utf-8")
+        config_variables_main = self.__get_configuration_main()
+        config_variables = self.__get_configuration_extra(config_variables_main)
+        return config_variables_main | config_variables
 
-            # If the file is empty, the YAML library will load as `None`. Since
-            # we expect this function to return a valid dict, we return an
-            # empty dictionary if it's empty.
-            yaml_data = self.yaml.load(raw_data)
-            if yaml_data is None:
-                return {}
-            return yaml_data
+    def __get_configuration_main(self) -> Dict:
```

Comment:
Prefer if we called it `_get_main_configuration`, easier to understand what it's getting straight away

'Private' functions in Python should be using a single `_` to avoid confusion with `__` dunder functions. We might not be consistent with that advice throughout `appcli`, but we should try to fix it as we go

Reply:
Fixed this one, along with the `_get_extra_configuration()` method.
Also updated the rest of the private methods in `VariableManager` to use single underscore.